packages/headless/src/features/facets/range-facets/date-facet-set/date-facet-selectors.ts

```diff
@@ -35,3 +35,14 @@ export const dateFacetSelectedValuesSelector = (
   }
   return facetResponse.values.filter((value) => value.state === 'selected');
 };
+
+export const dateFacetExcludedValuesSelector = (
+  state: SearchSection & DateFacetSection,
+  facetId: string
+): DateFacetValue[] => {
+  const facetResponse = dateFacetResponseSelector(state, facetId);
+  if (!facetResponse) {
+    return [];
+  }
```

Comment:
```suggestion
  const facetResponse = dateFacetResponseSelector(state, facetId) || [];
```

Reply:
I like this idea, but it wouldn't _quite_ work exactly as suggested because the filtering logic below depends on the `values` property of this response. Because of this, it would either have to be
```
const facetResponse = dateFacetResponseSelector(state, facetId) ?? {
    values: [],
  };
```

OR update all of the logic within this method to

```
const facetResponseValues =
    dateFacetResponseSelector(state, facetId)?.values ?? [];
  return facetResponseValues.filter((value) => value.state === 'selected');
```

I think I personally prefer the first one 🙂